crawler_mcp/core/vectors/statistics.py

```
@@ -0,0 +1,375 @@
+logger = logging.getLogger(__name__)
+
+
+class StatisticsCollector(BaseVectorService):
+    """
+    Collects and analyzes vector database statistics and source information.
+    """
+
+    def __init__(self, client: AsyncQdrantClient | None = None) -> None:
+        """
+        Initialize the statistics collector.
+        
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)